DependencyInjection/SonataBlockExtension.php

```diff
@@ -72,7 +72,9 @@ public function load(array $configs, ContainerBuilder $container)
         $this->configureProfiler($container, $config);
         $this->configureException($container, $config);
         $this->configureMenus($container, $config);
-        $this->configureClassesToCompile();
+        if (\PHP_VERSION_ID < 70000) {
```

Comment:
I personally prefer using the constant:
 - It's what symfony uses in it's codebase
 - The php optimizer to can optimize it and remove this part of code on the corresponding php versions